lib/TemplatedPathPlugin.js

```diff
@@ -111,6 +112,8 @@ const replacePathVariables = (path, data) => {
 			.replace(REGEXP_FILEBASE, getReplacer(data.basename))
 			// query is optional, it's OK if it's in a path but there's nothing to replace it with
 			.replace(REGEXP_QUERY, getReplacer(data.query, true))
+			// only available in sourceMappingURLComment
+			.replace(REGEXP_URL, getReplacer(data.url, true))
```

Comment:
```suggestion
			.replace(REGEXP_URL, getReplacer(data.url))
```

Reply:
Thanks for the suggestion! That param `[url]` should be intended and carefully defined.